src/extension/tab/helpers.ts

```diff
@@ -92,38 +146,69 @@ export function getQueriesLegacy(
       document: DocumentNode;
       variables: Variables;
       diff: Cache.DiffResult<any>;
+      networkStatus?: NetworkStatus;
     }
   >
 ): QueryInfo[] {
   let queries: QueryInfo[] = [];
   if (queryMap) {
-    queries = [...queryMap.values()].map(({ document, variables, diff }) => ({
-      document,
-      variables,
-      cachedData: diff?.result,
-    }));
+    queries = [...queryMap.values()].map(
+      ({ document, variables, diff, networkStatus }) => ({
+        document,
+        variables,
+        cachedData: diff?.result,
+        networkStatus,
+      })
+    );
   }
   return queries;
 }
 
+interface MutationStoreValue {
+  mutation: DocumentNode;
+  variables: Variables;
+  loading: boolean;
+  error: Error | null;
+}
+
 export function getMutations(
-  mutationsObj: Record<string, { mutation: DocumentNode; variables: Variables }>
-): QueryInfo[] {
+  mutationsObj: Record<string, MutationStoreValue>
+): MutationInfo[] {
   const keys = Object.keys(mutationsObj);
 
   if (keys.length === 0) {
     return [];
   }
 
   return keys.map((key) => {
-    const { mutation, variables } = mutationsObj[key];
+    const { mutation, variables, loading, error } = mutationsObj[key];
     return {
       document: mutation,
       variables,
+      loading,
+      error: getSerializedMutationError(error),
     };
   });
 }
 
+function serializeError(error: Error) {
```

Comment:
is `Error` an assumption we can safely make here? :thinking:

Reply:
Paired and added with [fd6a804](https://github.com/apollographql/apollo-client-devtools/pull/1429/commits/fd6a80451ee8829641666347f8bf3c7fb51bc1fc)